common/CoolMount.cpp

```diff
@@ -310,6 +310,7 @@ int domount(int argc, const char* const* argv)
         {
             // Now we need to set read-only and other flags with a remount.
             unsigned long mountflags = (MS_BIND | MS_REMOUNT | MS_NODEV | MS_NOSUID | MS_RDONLY);
+            const char* fstype = "none";
```

Comment:
The ... "filesystemtype arguments are ignored", hopefully there isn't some annoying apparmor rule that will kick in with an unknown fstype.

Reply:
Of course. This is to please the pedantic valgrind's check, which is bogus.